themes/dark/templates/addShows_addExistingShow.mako

```diff
@@ -185,21 +190,26 @@ window.app = new Vue({
                     console.log('Response: ' + message);
                 }
             }
+
             if (redirect) {
                 const baseUrl = apiRoute.defaults.baseURL;
-                if (params.length === 0) {
+                if (shows_to_add.length === 0) {
                     window.location.href = baseUrl + redirect;
                     return;
                 }
 
                 const form = document.createElement('form');
```

Comment:
Is the form needed at all anymore? You should be able to use `api.post` for this.